central/hash/datastore/datastore.go

```diff
@@ -39,3 +43,9 @@ func (d *datastoreImpl) GetHashes(ctx context.Context, clusterID string) (*stora
 func (d *datastoreImpl) DeleteHashes(ctx context.Context, clusterID string) error {
 	return d.store.Delete(ctx, clusterID)
 }
+
+// TruncateHashes truncates the hash table
+func (d *datastoreImpl) TruncateHashes(ctx context.Context) error {
+	_, err := d.db.Exec(ctx, "TRUNCATE TABLE hashes")
```

Comment:
[nit] Could we use the `HashesTableName` constant from the table schema here?

Reply:
On that note, it's a good idea to specify a schema as well, because generally speaking the default schema could be changed.